.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Report a bug
+about: Report a bug which you have noticed, to help us improve Neurolab
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Actual Behaviour**
+<!--A clear and concise description of the existing behaviour-->
+
+**Expected Behaviour**
+<!--A clear and concise description of what you want to happen.-->
+
+**Steps to reproduce it**
+<!--Add steps to reproduce the bug.-->
+
+**LogCat for the issue**
+<!--Provide logs for the crash here.-->
+
+**Screenshots of the issue**
+<!--Where-ever possible add a screenshot of the issue.-->
+
+-   Device: (e.g. iPhone6)
+-   OS: (e.g. iOS8.1)
+-   Flutter version: 
+
```

Comment:
Add section for Additional Context about the Bug

Reply:
Yes, my bad. I did some code changes in previous pull requests, in this one just specific issue fix as needed.